@coderabbitai まだ通っていないテストがあります。解消するためにはどのようにすればいいですか？
```
Run bin/rails db:test:prepare test test:system
  bin/rails db:test:prepare test test:system
  shell: /usr/bin/bash -e {0}
  env:
    RAILS_ENV: test
Running 28 tests in a single process (parallelization threshold is 50)
Run options: --seed 6036

# Running:

E

Error:
TurbidityToBodsControllerTest#test_authenticated_user_can_get_index:
StandardError: No fixture named 'one' found for fixture set 'users'
    test/controllers/turbidity_to_bods_controller_test.rb:5:in 'block in <class:TurbidityToBodsControllerTest>'

bin/rails test test/controllers/turbidity_to_bods_controller_test.rb:8

E

Error:
TurbidityToBodsControllerTest#test_unauthenticated_user_cannot_get_index:
StandardError: No fixture named 'one' found for fixture set 'users'
    test/controllers/turbidity_to_bods_controller_test.rb:5:in 'block in <class:TurbidityToBodsControllerTest>'

bin/rails test test/controllers/turbidity_to_bods_controller_test.rb:16

......E

Error:
UserAuthenticationFlowTest#test_signin_with_consumer_user:
StandardError: No fixture named 'two' found for fixture set 'users'
    test/integration/user_authentication_flow_test.rb:61:in 'block in <class:UserAuthenticationFlowTest>'

bin/rails test test/integration/user_authentication_flow_test.rb:60

F

Failure:
UserAuthenticationFlowTest#test_signin_with_producer_user [test/integration/user_authentication_flow_test.rb:87]:
Expected response to be a redirect to <http://www.example.com/> but was a redirect to <http://www.example.com/signin>.
Expected "http://www.example.com/" to be === "http://www.example.com/signin".

bin/rails test test/integration/user_authentication_flow_test.rb:79

.....F

Failure:
UsersControllerTest#test_should_create_user [test/controllers/users_controller_test.rb:19]:
`User.count` didn't change by 1, but by 0.
Expected: 4
  Actual: 3

bin/rails test test/controllers/users_controller_test.rb:18

..........F

Failure:
ReviewsControllerTest#test_should_get_new [test/controllers/reviews_controller_test.rb:15]:
Expected response to be a <2XX: success>, but was a <302: Found> redirect to <http://www.example.com/signin>
Response body: 

bin/rails test test/controllers/reviews_controller_test.rb:13

.

Finished in 4.252500s, 6.5844 runs/s, 20.9289 assertions/s.
28 runs, 89 assertions, 3 failures, 3 errors, 0 skips
Error: Process completed with exit code 1.
```

--- a/app/models/review.rb
+++ b/app/models/review.rb
@@ -11,7 +11,7 @@ class Review < ApplicationRecord
   private
 
   def reviewer_must_be_consumer
-    unless reviewer.is_consumer || !reviewee.is_consumer
+    unless reviewer.is_consumer && !reviewee.is_consumer
       errors.add(:reviewee, "Customer can only review producers")
     end
   end

--- a/app/views/layouts/application.html.erb
+++ b/app/views/layouts/application.html.erb
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="ja">
   <head>
     <title><%= content_for(:title) || "UNDO" %></title>
     <meta name="viewport" content="width=device-width,initial-scale=1">

--- a/app/views/layouts/mailer.html.erb
+++ b/app/views/layouts/mailer.html.erb
@@ -1,12 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html lang="ja">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <style>
       /* Email styles need to be inline */
     </style>
   </head>
-
   <body>
     <%= yield %>
   </body>

--- a/app/views/layouts/main.html.erb
+++ b/app/views/layouts/main.html.erb
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="ja">
   <head>
     <title><%= content_for(:title) || "UNDO" %></title>
     <meta name="viewport" content="width=device-width,initial-scale=1">

--- a/test/controllers/reviews_controller_test.rb
+++ b/test/controllers/reviews_controller_test.rb
@@ -3,6 +3,11 @@ require "test_helper"
 class ReviewsControllerTest < ActionDispatch::IntegrationTest
   setup do
     @review = reviews(:one)
+    @user = users(:carol)
+    post signin_path, params: {
+      email: @user.email,
+      password: "passwordcarol"
+    }
   end
 
   test "should get index" do

--- a/test/integration/user_authentication_flow_test.rb
+++ b/test/integration/user_authentication_flow_test.rb
@@ -58,12 +58,12 @@ class UserAuthenticationFlowTest < ActionDispatch::IntegrationTest
   end
 
   test "signin with consumer user" do
-    consumer_user = users(:two)
+    consumer_user = users(:carol)  # users(:two) から変更
     assert consumer_user.is_consumer
 
     post signin_path, params: {
       email: consumer_user.email,
-      password: "password"
+      password: "passwordcarol"  # "password" から変更
     }
     assert_redirected_to root_url
 
@@ -82,7 +82,7 @@ class UserAuthenticationFlowTest < ActionDispatch::IntegrationTest
 
     post signin_path, params: {
       email: producer_user.email,
-      password: "password"
+      password: "passwordbob"
     }
     assert_redirected_to root_url
 

